web/ui/templates/targets.html

```diff
@@ -54,5 +54,20 @@ <h2 id="targets">Targets</h2>
           </tbody>
         {{end}}
       </table>
+
+    <h2 id="alertmanagers">Alertmanagers</h2>
```

Comment:
We already have 5 different status pages. I feel that it's getting a bit too much.

How about adding it to the runtime information?